documentation/docs/backend/testing.md

```diff
@@ -0,0 +1,3 @@
+# Testing
+
+Django automated tests have been made to test all our endpoints. Unit tests can be run from within the backend Docker container using the command `python manage.py test` or you can directly execute it into the docker container `docker exec -it csf_server python manage.py test`.
```

Comment:
Is this saying that the tests can be run by first entering the container using `docker exec -it csf_server` followed by `python manage.py test` or the tests can be directly run inside the container using `docker exec -it csf_server python manage.py test`?

Reply:
Should this then just say the tests can be run inside the backend Docker container using `docker exec -it csf_server python manage.py test`?

Reply:
I've given two options on how to run the testing.

Reply:
Ok no worries.